packages/web-runtime/src/components/Topbar/ApplicationsMenu.vue

```diff
@@ -66,6 +71,27 @@ export default {
       onHidden: () => this.$refs.menubutton.$el.focus(),
       onShown: () => this.$refs.menu.$el.querySelector('a:first-of-type').focus()
     })
+  },
+  methods: {
+    async clickApp(appEntry) {
+      // @TODO use id or similar
+      if (appEntry.iconMaterial === 'switch_ui') {
```

Comment:
as discussed, let's go for `appEntry.url?.endsWith('/apps/files')` for the time being. That seems to be more reliable than an icon prop for now...